broker/utils/linkedlist.py

```diff
@@ -0,0 +1,126 @@
+# Author: Javan Lacerda - UFCG - LSD - 2019
+
+
+class LinkedList():
```

Comment:
LinkedList is a bad name for this class, since this, besides the usual linked list behavior, manages the remaining time calculation

Reply:
Solved. I created a new class that extend the LinkedList and extract the insert behavior for this new class.